### ISSUE TITLE
Blueprint spamming logbook

### ISSUE THREAD

**Issue by ra-kal** [Blackymas/NSPanel_HA_Blueprint#1244]
Hello,

I noticed the Blueprint automation spams logbook with a record every second. I'm currently using devel version 4.1dev3, so I'm not sure if this is intended behaviour. Anyway, it makes the logbook pretty unusable and probably the database will be populated with the logs.

**Comment by edwardtfn** [comment]
This is not intentional. The logbook registers every time an automation is triggered (and conditions are all satisfied), and we don´t have control over this, although you can disable your logbook for this specific automation with somethin like this:

```yaml
logbook:
  exclude:
    entities:
      - automation.your_nspanel_configuration
```
Source: https://www.home-assistant.io/integrations/logbook/

This, however, will eliminate all entries in the logbook related to that automation.

But I would instead try to understand why your automation is triggering every second. Usually this happens if you have some of the values or chips shown in the Home page connected to an entity which has an update very frequently.
Could that be the case?

The Home page is the only one which will be updated while not visible (even when the panel is sleeping) and this was introduced on v3.1 (I think) with a try to improve performance when displaying that page.
The other pages will be updated only when visible.

**Comment by ra-kal** [comment]
Thanks for the clarification. It was not clear to me, because it was just logged as triggered by event "state_canged".
In my case it was caused by button showing media_player icon on the home page. Even though it only shows the icon state, it spams the logbook with all changes (current movie position).

**Comment by edwardtfn** [comment]
mmmm...
Good to know that. I probably can find a way to prevent this to trigger the automation.

**Comment by edwardtfn** [comment]
I believe I have a fix for this, now on `dev`. Hopefully it will be included in the next release, but if you have a chance to test it earlier on `dev`, please let me know.

**Comment by Kepro** [comment]
> But I would instead try to understand why your automation is triggering every second.

In my case it is solar production & grid consumption so it is pretty "live"

**Comment by edwardtfn** [comment]
Which version are you running?

**Comment by edwardtfn** [comment]
I believe all changes made for this are included in v4.1.4, which was released earlier last month.

I'm not sure if we can do much more here. With the last changes, those values are updated only when Home page is visible, but if you have that page visible, each change on the source data will trigger an update to that component on the page, which includes it's icon and value.

**Comment by pkasarda** [comment]
I'm using NSPanel on desktop to monitor heat pump and solar production :-D so display is not dimmed, but yeah disabled logs fixed it... I think that is only downside from using "automation" as solution :-) which I love
